device-communication/pom.xml

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.eclipse.hono</groupId>
+    <artifactId>device-communication</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <properties>
+        <compiler-plugin.version>3.10.1</compiler-plugin.version>
+        <maven.compiler.release>17</maven.compiler.release>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+        <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+        <quarkus.platform.version>2.15.3.Final</quarkus.platform.version>
+        <skipITs>true</skipITs>
+        <surefire-plugin.version>3.0.0-M7</surefire-plugin.version>
+        <org.mapstruct.version>1.5.3.Final</org.mapstruct.version>
+        <hono.version>2.1.0</hono.version>
+    </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>${quarkus.platform.group-id}</groupId>
+                <artifactId>${quarkus.platform.artifact-id}</artifactId>
+                <version>${quarkus.platform.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc</artifactId>
```

Comment:
`quarkus-arc` includes a dependency of `quarkus-ide-launcher`. In Hono, we have excluded the `quarkus-ide-launcher` dependency - it is marked by the eclipse license check tool as not having consistent license information and it didn't seem necessary. Do we need that dependency here? (I tested running the application with Intellij IDEA Ultimate via a Quarkus run configuration - that worked without the dependency.) If not, we could add an exclusion here:
```
            <exclusions>
                <exclusion>
                    <groupId>io.quarkus</groupId>
                    <artifactId>quarkus-ide-launcher</artifactId>
                </exclusion>
            </exclusions>
```
This should then also be added to the `quarkus-vertx` dependency declaration below.

Reply:
Done.